cyral/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Cyral",
+  "maintainer": "product@cyral.com",
+  "manifest_version": "1.0.0",
+  "name": "cyral",
+  "metric_prefix": "cyral.",
+  "metric_to_check": "cyral.analysis_time",
+  "creates_events": false,
+  "short_description": "Collect runtime metrics from a Cyral instance monitoring MySQL.",
+  "guid": "2a854a73-b0da-4954-b34e-fc1cd05ba8e8",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Cyral",
+  "categories": [
+    "monitoring",
+    "security"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "cyral",
+  "assets": {
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
+    "dashboards": {},
```

Comment:
We'll also need a default dashboard included in the integration.

```
    "dashboards": {
      "Cyral Overview": "assets/dashboards/cyral_overview.json"
    },
```

Reply:
Please refer to our dashboard guidelines on style and how to export a dashboard: https://datadoghq.dev/integrations-core/guidelines/dashboards/

Reply:
Thanks! that's very helpful.